examples/nlp/language_modeling/megatron_change_num_partitions.py

```diff
@@ -199,7 +199,7 @@ def compute_tp_splits(
     # alias the global index to idx
     idx = global_idx
 
-    swiglu_activation = 'swiglu' in str(model_cfg.get('activation', '')).lower()
+    fast_glu_activation = str(model_cfg.get('activation', '')).lower() in ['fast-geglu', 'fast-swiglu', 'fast-reglu']
```

Comment:
Add swiglu to the list

Reply:
`swiglu` didn't use the torch chunk tricks, so we don't need to handle the partition. Only `fast_glu_activation` need. 
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/modules/common/megatron/mlp.py#L230-L231

Reply:
From `Megatron-LM`, they do special handing for `swiglu` for partition conversion ([code](https://github.com/NVIDIA/Megatron-LM/blob/060415572f4365a2e895f8036c4e37dad0efbdf5/tools/checkpoint_saver_megatron.py#L277-L281)) is because they have chunk operation for `swiglu` in their implementation ([code](https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/model/transformer.py#L115-L119)). In NeMo, we also have chunk operation  but it only used when our activation is `fast_glu_activation` ([code](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/modules/common/megatron/mlp.py#L230-L231)). Therefore, I change the partition conversion script from `swiglu` to `fast_glu_activation`

Reply:
Ok makes sense.